zulipterminal/ui_tools/views.py

```diff
@@ -273,7 +273,7 @@ def update_streams(self, search_box: Any, new_text: str) -> None:
             streams_display = [
                 stream
                 for stream in self.streams_btn_list.copy()
-                if stream.stream_name.lower().startswith(new_text.lower())
+                if new_text.lower() in stream.stream_name.lower()
```

Comment:
You could hoist the repeated `lower` operation out of the loop? (in both places)